src/BlazorUI/Bit.BlazorUI/Components/Inputs/Rating/BitRating.scss

```
@@ -1,9 +1,23 @@
-﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/functions.scss";
 
 .bit-rtg {
     font-weight: 400;
+    width: fit-content;
     font-size: spacing(1.75);
     font-family: $tg-font-family;
+    // The unfilled part stays neutral whatever the Color is, so it keeps reading as "not rated yet"
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)